internal/gatewayapi/clienttrafficpolicy.go

```diff
@@ -521,19 +525,34 @@ func translatePathSettings(pathSettings *egv1a1.PathSettings, httpIR *ir.HTTPLis
 	}
 }
 
-func translateClientTimeout(clientTimeout *egv1a1.ClientTimeout, httpIR *ir.HTTPListener) error {
+func buildClientTimeout(clientTimeout *egv1a1.ClientTimeout) (*ir.ClientTimeout, error) {
```

Comment:
which func name prefix do we prefer here? I see we are using different prefix names for these func, like `buildXXX` and `translateXXX`?

Is a non blocking comment, but IMO we should unify them.

Reply:
vote for `build`, it's consistent with all the build methods in BTP